src/wizard/deploy.jl

```diff
@@ -306,11 +306,20 @@ function common_git_repo_setup(repo_dir, repo, state)
         break
     end
 
+    # Helper function to get around LibGit2's strange `get()` behavior
+    function lg2get(T, cfg, name)
+        try
+            return LibGit2.get(T, cfg, name)
+        catch
```

Comment:
Perhaps worth checking the type of the exception or something? In case someone touches this later and introduces an `UndefVarError`.